FindYourRecipe/FindYourRecipe.Api/Controllers/CategoryRecipeController.cs

```diff
@@ -1,15 +1,12 @@
-﻿using System;
-using FindYourRecipe.Application;
-using FindYourRecipe.Application.Interfaces;
-using FindYourRecipe.Application.Models;
-using FindYourRecipe.DataAccess.Interfaces;
-using FindYourRecipe.DataAccess.Repositories;
+﻿using FindYourRecipe.Application;
+using FindYourRecipe.Contracts;
+using FindYourRecipe.Contracts.Models;
 using Microsoft.AspNetCore.Mvc;
 
 namespace FindYourRecipe.Api.Controllers
 {
 	[ApiController]
-	[Route("[controller]")]
+	[Route("category-recipes")]
```

Comment:
I suggest using `categoryRecipes` instead of the dash `-`. It is not wrong to have the dash in there but I haven't seen it widely used. Usually camel case is the preferred option (see https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/ for more details about camel case).